### PR TITLE
Providing support for bzip2 compressed assets

### PR DESCRIPTION
**Pull request by xiam**
This PR provides support for reading bzip2 compressed assets on the update server. The Makefile was also modified in order to generate such assets.

### CODE REVIEW

**Comment by xiam** [comment]
Hello @atavism, you helped me reviewing the auto-update PR, could you help me taking a look at this too?

Thanks,

**Comment by atavism** [comment]
@xiam Sure!

File: src/github.com/getlantern/autoupdate-server/server/asset.go [hunk 1]
```diff
@@ -46,6 +49,10 @@ func downloadAsset(uri string) (localfile string, err error) {
 			return "", fmt.Errorf("Expecting 200 OK, got: %s", res.Status)
 		}
 
+		if res.Body != nil {
+			defer res.Body.Close()
+		}
+
```
Comment (atavism) [inline src/github.com/getlantern/autoupdate-server/server/asset.go:53]: If http.Get doesn't return an error, do we need to check the response body for nil here?

Comment (xiam) [inline src/github.com/getlantern/autoupdate-server/server/asset.go:53]: > When err is nil, resp always contains a non-nil resp.Body. Caller should close resp.Body when done reading from it.

Right @atavism! this is not really required. Fixed.

**Comment by atavism** [comment]
LGTM

**Pull request merged**